modules/core/res/META-INF/services/edu.kit.iti.algover.term.prettyprint.PrettyPrintExtension

```diff
@@ -7,4 +7,4 @@ edu.kit.iti.algover.term.prettyprint.HeapStorePrinterExtension
 edu.kit.iti.algover.term.prettyprint.SequencePrinterExtension
 edu.kit.iti.algover.term.prettyprint.ExtensionsPrinterExtension
 edu.kit.iti.algover.term.prettyprint.DafnyFunctionPrinterExtension
-edu.kit.iti.algover.term.prettyprint.SubscriptPrinterExtension
```

Comment:
This class can probably be removed then.

Reply:
Yes.